src/dev/src/app/tabs/tabs-angular.ts

```diff
@@ -4,12 +4,14 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component } from '@angular/core';
+import { TabsLayout } from '@clr/angular';
 
 @Component({
   selector: 'clr-modal-tabs-angular',
   styleUrls: ['./tabs.demo.scss'],
   templateUrl: './tabs-angular.demo.html',
 })
 export class TabsAngularDemo {
+  layout: TabsLayout = TabsLayout.VERTICAL;
```

Comment:
We really don't want to use the `TabsLayout` in our demos directly, so this should just be the string value.

Reply:
Not using TabsLayout makes things a lot simpler - removing!

And about the tabLinkDirectives.first, yes - it is cooler, but tabLinkDirectives is not a QueryList anymore, but an array, so it has to be [0] instead. We can't use the QueryList returned by the previous @ContentChildren (descendants: true), because it also collects child nodes of nested tabs.